examples/enum_in_args.rs

```diff
@@ -1,16 +1,57 @@
 #[macro_use]
 extern crate structopt;
-#[macro_use]
-extern crate clap;
 
 use structopt::StructOpt;
 
-arg_enum! {
-    #[derive(Debug)]
-    enum Baz {
-        Foo,
-        Bar,
-        FooBar
+use std::ascii::AsciiExt;
+use std::fmt;
+use std::str::FromStr;
+
+// Equivalents to
+// ```
+// clap::arg_enum! {
+//     #[derive(Debug)]
+//     enum Baz {
+//         Foo,
+//         Bar,
+//         FooBar,
+//     }
+// }
+//```
```

Comment:
I don't want to remove this example, a lot of person look at it. If that's a problem about the minimum rust version, as clap has updated its version, we can as described in the policy.

Reply:
<https://travis-ci.org/TeXitoi/structopt/jobs/519234259#L465>
I had thought `tests` compiled and executed `examples`. We don't need to modify `enum_in_args` because we can avoid the Travis error by adding just `--tests` option for `1.21.0` to skip compiling `examples/`. Though [`clap` has already stopped supporting `1.21.0`](https://github.com/clap-rs/clap/pull/1397), I cannot think of any other reason to bump the Rust version. `structopt` and its tests work for now.